tests/unittest/_torch/auto_deploy/unit/singlegpu/transformations/library/test_quantization.py

```
@@ -113,6 +113,51 @@ def test_quantization(quant_config, atol, rtol, num_p_og):
     torch_export_to_gm(gm_transformed, args=(x,))
 
 
+@pytest.mark.parametrize("bias", [True, False])
+@pytest.mark.skipif(not fp8_compatible(), reason="Requires fp8 support")
+def test_finegrained_fp8_quantization(bias):
```

Ruff (0.15.1)

[warning] Unused function argument: `bias`

(ARG001)